### PR TITLE
[web-components] update components to extend foundation element

### PR DESCRIPTION
**Pull request by chrisdholt**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [x] Include a change request file using `$ yarn change`

#### Description of changes
Updates web components to extend foundation element. This is the first of several steps for our first major version for the Fluent Web Components
#### Focus areas to test

(optional)

### CODE REVIEW

**Review by eljefe223 — APPROVED** [review]
Smoked, compiles with no errors.

**Review by awentzel — APPROVED** [review]

File: packages/web-components/src/index-rollup.ts [hunk 1]
```diff
@@ -1,7 +1,13 @@
...
-export * from '@microsoft/fast-element';
-export * from '@microsoft/fast-foundation';
 
-// Re-export Design system to avoid conflicts with the new class from FAST Foundation
-// TODO: Update name to FluentDesignSystem once DI is implemented and published from FAST
-export { DesignSystem, DesignSystemDefaults } from './fluent-design-system';
+/**
+ * TODO rename this to FASTDesignSystem when {@link @FASTDesignSystem} interface is removed.
+ */
+export const fastDesignSystem = DesignSystem.getOrCreate().register(
```
Comment (EisenbergEffect) [inline packages/web-components/src/index-rollup.ts:11]:
```suggestion
export const fluentDesignSystem = DesignSystem.getOrCreate().register(
```

File: packages/web-components/src/index-rollup.ts [hunk 2]
```diff
@@ -1,7 +1,13 @@
...
+
 export * from './index';
-export * from '@microsoft/fast-element';
-export * from '@microsoft/fast-foundation';
 
-// Re-export Design system to avoid conflicts with the new class from FAST Foundation
-// TODO: Update name to FluentDesignSystem once DI is implemented and published from FAST
-export { DesignSystem, DesignSystemDefaults } from './fluent-design-system';
+/**
+ * TODO rename this to FASTDesignSystem when {@link @FASTDesignSystem} interface is removed.
```
Comment (EisenbergEffect) [inline packages/web-components/src/index-rollup.ts:9]:
```suggestion
 * TODO rename this to FluentDesignSystem when {@link @FluentDesignSystem} interface is removed.
```

**Review by EisenbergEffect — APPROVED** [review]
A couple minor suggestions but otherwise looks good for phase 1.

**Review by nicholasrice — APPROVED** [review]

**Pull request merged**